modules/thunder_article/modules/thunder_fia/templates/user--facebook-instant-articles-rss.html.twig

```diff
@@ -0,0 +1 @@
+<a>{{ content.field_forename|render|trim }} {{ content.field_surname|render|trim }}</a>
```

Comment:
There are no field_forename and field_surname fields in Thunder. Please use just the username